server/controllers/routes/user/student/getAlumniCohort.js

```diff
@@ -0,0 +1,28 @@
+const { getSpecificAlumni } = require('../../../../database/queries');
+const { getCohortQuery } = require('../../../../database/queries');
+
+const getAlumniCohort = async (req, res, next) => {
+  try {
+    const { cohortId } = req.params;
+    if (cohortId > 0) {
```

Comment:
what if the cohortId is not a number?
maybe you can convert it to a number then you can check it
or use isNaN function